osu.Game.Rulesets.Mania/Objects/Drawables/DrawableNote.cs

```diff
@@ -38,6 +41,9 @@ public partial class DrawableNote : DrawableManiaHitObject<Note>, IKeyBindingHan
 
         private Drawable headPiece;
 
+        [CanBeNull]
+        private DrawablePerfectBonusNote bonusNote;
```

Comment:
Why's this nullable? Is this a leftover from the version where the bonus thingy was added conditionally? Right now, as far as I can tell, this should be there on all objects...

Reply:
It was just a safety. I'll remove it for now.